values.yaml

```diff
@@ -941,6 +941,51 @@ ui:
     # @type: string
     additionalSpec: null
 
+  # Configure Ingress for the Consul UI.
+  ingress:
+    # This will create an Ingress resource for the Consul UI.
+    # @type: boolean
+    enabled: false
+
+    # hosts is a list of host name to create Ingress rules.
+    # The value below is an array of objects, examples are shown below.
+    #
+    # ```yaml
+    # hosts:
+    #   - host: foo.bar
+    #     paths:
+    #       - /example
+    #       - /test
+    # ```
+    #
+    # @type: array<map>
+    hosts: []
+
+    # tls is a list of hosts and secret name in an Ingress
+    # which tells the Ingress controller to secure the channel.
```

Comment:
Should we tell them what secret to use? Do they need to use passthrough TLS if consul tls is enabled?

Reply:
So i realized in my tinkering that i hadn't tested the only TLS (https only enabled) options. ie, the 80 port was always open on the service. the ingress resource always performs TLS termination so we cant point traffic to 443 as the request that reached the service will always be HTTP. I guess it doesnt make sense for this PR to support port 443 in that case, and ill make the requisite changes.